classes/module/Module.php

```diff
@@ -2047,6 +2047,34 @@ public function displayConfirmation($string)
         </div>';
         return $output;
     }
+    
+    /**
+    * Helper displaying information message
+    * @param string $string|array
+    * @return string
+    */
+    public function displayInformation($string)
```

Comment:
Be careful about the PHP doc and the variable name. `$string` is not anymore a proper name depending on its type.

    /**
     * Helper displaying information message
     * @param string|array $information
     * @return string
     */
    public function displayInformation($information)
    {
        // [...]
```

Reply:
Yes but it is not the case in displayConfirmation